src/index.html

```diff
@@ -1,12 +1,323 @@
+      <article class="card" data-qa="card">
+        <div class="card__img-container">
+          <img
+            src="./images/imac.jpeg"
+            class="card__img"
+            alt="picture"
+          >
+        </div>
+
+        <h3 class="card__description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+
+        <p class="card__product">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p>Reviews: 5</p>
+        </div>
+
+        <div class="card__tag">
+          <p class="card__price">Price:</p>
+          <p class="card__value">$2,199</p>
+        </div>
+
+        <a href="#" data-qa="card-hover">
+          <p class="card__button">Buy</p>
+        </a>
+      </article>
+      <article class="card" data-qa="card">
+        <div class="card__img-container">
+          <img
+             src="./images/imac.jpeg"
+             class="card__img"
+             alt="picture"
+          >
+        </div>
+
+        <h3 class="card__description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+
+        <p class="card__product">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p>Reviews: 5</p>
+        </div>
+
+        <div class="card__tag">
+          <p class="card__price">Price:</p>
+          <p class="card__value">$2,199</p>
+        </div>
+
+        <a href="#" data-qa="card-hover">
+          <p class="card__button">Buy</p>
+        </a>
+      </article>
+      <article class="card" data-qa="card">
+        <div class="card__img-container">
+          <img
+             src="./images/imac.jpeg"
+             class="card__img"
+             alt="picture"
+          >
+        </div>
+
+        <h3 class="card__description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+
+        <p class="card__product">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p>Reviews: 5</p>
+        </div>
+
+        <div class="card__tag">
+          <p class="card__price">Price:</p>
+          <p class="card__value">$2,199</p>
+        </div>
+
+        <a href="#" data-qa="card-hover">
```

Comment:
add data-qa="card-hover" (not just hover) to the link Buy only inside the first card